packages/local_auth/local_auth/CHANGELOG.md

```diff
@@ -1,9 +1,10 @@
-## NEXT
+## 2.2.0
 
-* Updates minimum supported SDK version to Flutter 3.13/Dart 3.1.
+* Switches endorsed iOS implementation to `local_auth_darwin`.
+  * Clients directly importing `local_auth_ios` for auth strings should switch
+    dependencies and imports to `local_auth_darwin`. No other change is necessary.
 * Updates support matrix in README to indicate that iOS 11 is no longer supported.
-* Clients on versions of Flutter that still support iOS 11 can continue to use this
```

Comment:
I removed this because it no longer makes sense; with 3.16.6 as the minimum, nobody supporting iOS 11 can use this version of `local_auth` anyway.